### ISSUE TITLE
[BITV]: Remove incorrect aria-labelledby attributes on calendar avaliability and assign ul contianer to have correct aria-label

### ISSUE THREAD

**Issue by emoral435** [nextcloud/server#43618]
Currently, the calander has wrong aria-label semantics on it. Instead of each `li` element having an aria-labelledby attr, the `ul`  container should have the aria-description

link to exact location in code here:  https://github.com/nextcloud/calendar-availability-vue/blob/cd16301c6f288331e1c05f6ef34f3aafe4577809/src/CalendarAvailability.vue#L2
